test/e2e/tests/test.spec.js

```diff
@@ -3,14 +3,18 @@ module.exports = {
   'Demo test' : function (browser) {
     browser
       .url(browser.launch_url)
+    browser
```

Comment:
Are they all atomic due to version `0.9`?

Reply:
I think I just played around because Browserstack didn't work as expected. It would execute a lot of the commands before the page was loaded.